app/client/layouts/main/main.html

```diff
@@ -1,7 +1,6 @@
 <template name="mainLayout">
   {{> mainLayoutNavbar }}
   <div class="container">
```

Comment:
@brylie I have removed flash messages here as all the messages are displaying twice in the app. Also I needed a message on the form in the modal.

Reply:
Make sure FlashMessages are still displaying in other places.

Reply:
It is being used in enroll uses, password change and add new feeling, and it is working fine.

Reply:
OK, thanks. We might consider deprecating this module at some point, so that all of the notifications use the same method.